docs/source/en/peft.md

```diff
@@ -125,7 +125,7 @@ Now you can use [`~peft.PeftModel.set_adapter`] to set which adapter to use:
 ```py
 # use adapter_1
 model.set_adapter("adapter_1")
-output = model.generate(**inputs)
+output_disabled = model.generate(**inputs)
```

Comment:
Are you sure about this one?

Reply:
https://github.com/huggingface/transformers/blob/a70286f827ea46573e498fbc185a2498bc67d709/docs/source/en/peft.md?plain=1#L126-L134

Yup, in here `output_disabled[0]` has been referenced but never declared in the file, and looking at the code below using the pretty much the same code block for `output_enabled`, I assume its typo, where `output` should have been `output_disabled`.